test/Header/ContentSecurityPolicyTest.php

```diff
@@ -139,4 +139,57 @@ public function testContentSecurityPolicySetDirectiveWithEmptyReportUriRemovesEx
             $csp->toString()
         );
     }
+
+    /**
+     * @param string $directive
+     * @param array $value
```

Comment:
What is the type of this `array`?

Reply:
Hi @Ocramius,
Changed to string[].

Thanks for your return :-)